tests/bluetooth/at/src/main.c

```diff
@@ -61,7 +61,7 @@ ZTEST(at_tests, test_at)
 
 	zassert_true(net_buf_tailroom(buf) >= len,
 		    "Allocated buffer is too small");
-	strncpy((char *)buf->data, example_data, len);
+	memcpy((char *)buf->data, example_data, len);
 	net_buf_add(buf, len);
```

Comment:
These two lines would actually be cleaner and simpler as a single function call:
```c
net_buf_add_mem(buf, example_data, len);
```

Reply:
@keith-packard could you still make this proposed change? Otherwise the PR looks good to me now.

Reply:
sure. having dug through the implementation of this stuff, that seems way more correct to me as well.